### PR TITLE
More stuff

### LINKED ISSUE #3: Docs: add table with amount of memory required at different matrix sizes
E.g. here's what I'm envisioning:

| Matrix Size                  | Required memory |
| --------------------- | ------------------ |
| 10 by 10                      | 123 MB                   |
| 100 by 100                 | 456 MB                   |
| 1k by 1k                      | 789 MB                   |
| 10k by 10k                  | 123 GB                   |
| 50k by 50k                 | 456 GB                  |
| 73k by 73k                  | 789 GB                  |
| 100k by 100k              | 123 TB                   |

I just made those numbers up - obviously they are incorrect. But it would be nice to have a table like that, with the actual numbers.

That way, when people want to run these benchmarks on their own computer, they can just:
1. Look up how much RAM their computer has
2. Cross-reference with this table to figure out the biggest matrix size they can use

Also, if someone wants to run the benchmarks on a cluster, they can ask their scheduler (e.g. SLURM) for the necessary amount of memory.

### PR DESCRIPTION
**Pull request by DilumAluthge**
Fixes #3

### CODE REVIEW

File: src/runbenchmark.jl [hunk 1]
```diff
@@ -102,10 +102,48 @@ Base.axes(ls::LogSpace) = axes(ls.r)
...
+             sizes = logspace(2, 4000, 200),
+             threaded::Bool = Threads.nthreads() > 1,
+             A_transform = identity,
+             B_transform = identity,
+             sleep_time = 0.0)
 """
 function runbench(
     ::Type{T} = Float64;
-    libs = [:BLIS, :Gaius, :MKL, :Octavian, :OpenBLAS, :Tullio],
+    libs = default_libs(),
```
Comment (chriselrod) [inline src/runbenchmark.jl:146]: `default_libs(T)`

File: src/plotting.jl [hunk 2]
```diff
@@ -47,6 +64,8 @@ function _plot(
     width::Real = 1200,
     height::Real = 600,
     plot_directory::AbstractString = default_plot_directory(),
+    plot_filename::AbstractString = default_plot_filename(),
+    file_extensions = ["svg", "png"],
```
Comment (chriselrod) [inline src/plotting.jl:68]: I like svg more for the vector graphics, and you can add them to docs. But you can't share them online, hence the pngs. So I think these are good defaults.

File: src/runbenchmark.jl (hunk 1 above)
Comment (DilumAluthge) [reply src/runbenchmark.jl]: Done!

**Comment by chriselrod** [comment]
Looks good, I'll merge when the tests pass.

**Comment by DilumAluthge** [comment]
@chriselrod Whew, this is finally green.

**Pull request merged**